tests/operator_test.go

```diff
@@ -420,7 +415,7 @@ var _ = Describe("ALL Operator tests", func() {
 			})
 
 			It("[test_id:4986]should remove/install CDI a number of times successfully", func() {
-				for i := 0; i < 10; i++ {
+				for i := 0; i < 5; i++ {
```

Comment:
Any reason to change this and the eventually timeout?

Reply:
I see no reason to make it sooo long. 5 iters seems more than enough I guess?